everestrs/Cargo.toml

```diff
@@ -2,4 +2,8 @@
 resolver = "2"
 members = [
    "everestrs",
+   "everestrs-build",
 ]
+
+[workspace.dependencies]
+everestrs-build = { path="everestrs-build" }
```

Comment:
Is there a reason why everestrs depends on everestrs-build and not the other way around?

Reply:
yes kind of - the build dependencies (read everestrs-build) are build before other dependencies (read everestrs)